test/e2e/framework/nodes_util.go

```diff
@@ -73,10 +73,19 @@ func etcdUpgradeGCE(target_storage, target_version string) error {
 	return err
 }
 
-func ingressUpgradeGCE() error {
+func ingressUpgradeGCE(mode bool) error {
 	// Flip glbc image from latest release image to HEAD to simulate an upgrade.
+	// Flip from HEAD to latest release image to simulate a downgrade.
 	// Kubelet should restart glbc automatically.
-	sshResult, err := NodeExec(GetMasterHost(), "sudo sed -i -re 's/(image:)(.*)/\\1 gcr.io\\/e2e-ingress-gce\\/ingress-gce-e2e-glbc-amd64:latest/' /etc/kubernetes/manifests/glbc.manifest")
+	var command string
+	if mode {
+		// Upgrade
+		command = "sudo sed -i -re 's/(image:)(.*)/\\1 gcr.io\\/k8s-ingress-image-push\\/ingress-gce-e2e-glbc-amd64:latest/' /etc/kubernetes/manifests/glbc.manifest"
+	} else {
+		// Downgrade
+		command = "sudo sed -i -re 's/(image:)(.*)/\\1 gcr.io\\/google_containers\\/glbc:0.9.7/' /etc/kubernetes/manifests/glbc.manifest"
```

Comment:
Is there a way to retrieve the latest released tag? Or we will have to update it every GLBC release.

Reply:
AFAIK, there is no way to retrieve the most recent image.

Reply:
If we tag each release with latest, the tags from previous releases (with tag "latest") will get removed.

Reply:
Looks like we can tag with "latest" and the version. I still want to experiment a little more to make sure doing this does not break anything. In the meantime, this should be fine.